tweakwcs/matchutils.py

```diff
@@ -22,12 +22,19 @@
 
 __author__ = 'Mihai Cara'
 
-__all__ = ['MatchCatalogs', 'XYXYMatch']
+__all__ = ['MatchCatalogs', 'XYXYMatch', 'MultiMatchError']
 
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
 
+class MultiMatchError(RuntimeError):
```

Comment:
I kind of do not like `MultiMatch` because it suggests that multiple matches are bad. Something like `MatchSourceConfusionError` seems more descriptive but maybe too long (maybe drop "source"?)

Reply:
Good point. I renamed this to `MatchSourceConfusionError` in https://github.com/spacetelescope/tweakwcs/pull/204/commits/dcaed934769f20b63172a051eebdc445c04ad987